.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sarif-viewer.connectToGithubCodeScanning": "off"
```

Comment:
Should we add this to `.gitignore`? And maybe the whole .vscode folder?

Reply:
done!